nengo_interfaces/airsim.py

```diff
@@ -253,6 +253,12 @@ def __init__(  # noqa: C901
                     )
                     # TODO
                     # dict.get(key, return val if missing)
+                    if "CameraDefaults" not in data.keys():
+                        data["CameraDefaults"] = cv_data["CameraDefaults"]
+                    elif "CaptureSettings" not in data["CameraDefaults"]:
+                        data["CameraDefaults"]["CaptureSettings"] = (
+                                cv_data["CameraDefaults"]["CaptureSettings"]
```

Comment:
It looks like this extra indent is causing the build to fail. Other than that, everything looks good to me!

Reply:
Pushed some rebased fixups, ci tests are passing now :+1: . If it looks good then go for the merge :partying_face:

Reply:
Pushed a rebased fixup, ci tests are passing now :partying_face: If it looks good to you, go for the merge :+1: